src/command_modules/azure-cli-iot/azure/cli/command_modules/iot/custom.py

```diff
@@ -150,7 +150,8 @@ def iot_hub_policy_create(client, hub_name, policy_name, permissions, resource_g
 def iot_hub_policy_delete(client, hub_name, policy_name, resource_group_name=None):
     hub = iot_hub_get(client, hub_name, resource_group_name)
     policies = iot_hub_policy_list(client, hub_name, hub.resourcegroup)
-    if not _is_policy_existed(policies, policy_name):
+    import copy
```

Comment:
Move import statement to the beginning of the current scope which this function.